src/main/java/jalau/at18/azul/view/FactoryButton.java

```diff
@@ -0,0 +1,23 @@
+package jalau.at18.azul.view;
+
+import javax.swing.JToggleButton;
+import java.awt.Color;
+
+public class FactoryButton extends JToggleButton {
```

Comment:
Maybe this class could be called `TileButton` instead of `FactoryButton` because these buttons are the ones that are part of the `TileButtonGroup`

Reply:
Already change the name of the classes, now the factory button it's tilesbutton and the tilebutton class now it's call bagbutton so it's no more confuse

Reply:
Hello team, i just noticed that "Controller" file and packages are with uppercase characters in its naming, it should be all lowercase like "controller" as stated in the group convention, please take that into account in the next PR